Makefile

```diff
@@ -114,6 +115,7 @@ resetdb: .state/docker-build-base
 .state/db-populated: .state/db-migrated
 	docker compose run --rm web python -m warehouse sponsors populate-db
 	docker compose run --rm web python -m warehouse classifiers sync
+	docker compose exec --user postgres db psql -U postgres warehouse -f /post-migrations.sql
```

Comment:
Should this be here as well as earlier? I feel like it should exist here once only - and probably remove this line.
```suggestion
```

Reply:
the earlier clause is for `make resetdb`, which needs to duplicate the call (at least how we're currently setup)

Reply:
okay. maybe when we tackle db seeds we can rethink a bunch of this.

Reply:
on closer inspection, you're right. the duplicate in `resetdb` can be removed!

Reply:
https://github.com/pypi/warehouse/pull/16257